change/@ni-nimble-blazor-ad0b7e2c-d4ff-4837-b0c0-ec203eee88f7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added logic to stop NimbleTabs from seeing `change` event from children components. Followed the pattern FluentUI Blazor uses for their Tab component.",
+  "packageName": "@ni/nimble-blazor",
+  "email": "tim.atwood@ni.com",
```

Comment:
The email field gets exposed to the public via the changelog so most of us have configured our machines to use the obfuscated github email. [Instructions available in CONTRIBUTING](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file). Once you do that you can manually modify the change file to use the github email or delete it and regenerate it after configuring your git user for the repo.

Reply:
Fixed in latest push.